src/stars.css

```diff
@@ -0,0 +1,91 @@
+.tutto {
+  height: 160px;
+  width: 140px;
+}
+
+.stars {
+  display: flex;
+  margin-left: 20px;
+  margin-right: 20px;
+}
```

Comment:
cannot set margin for blocks

Reply:
thanks.
in new version we don't have .stars at all. 
and I minimized using margins.